packages/cdk-dasm/test/__snapshots__/dasm.test.js.snap

```diff
@@ -0,0 +1,113 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`basic test 1`] = `
+"// generated by cdk-dasm at 2019-04-11T06:56:13.428Z
+
+import { Stack, StackProps, Construct, Fn } from '@aws-cdk/cdk';
+import sns = require('@aws-cdk/aws-sns');
+
+export class MyStack extends Stack {
+    constructor(scope: Construct, id: string, props: StackProps = {}) {
+        super(scope, id, props);
+        new sns.CfnTopic(this, 'MyTopic', {
+            displayName: \\"hello hello\\",
```

Comment:
These backslashes are not correct, are they?

Reply:
that's how jest serializes snapshots